README.md

```diff
@@ -54,49 +54,13 @@ To build the project:
 
 ### Linux
 
-Tested on a fresh **Ubuntu 16.04 LTS** VM installation (_equivalent processes should work for other distributions_)
+For building Inky on Linux, first install `npm` and `electron-packager`.
```

Comment:
Might want to mention mono in this comment too!

Reply:
Good point, it keeps coherence with the installation command